cmd/soroban-rpc/internal/test/docker-compose.rpc.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  rpc:
+    platform: linux/amd64
+    image: stellar/soroban-rpc:${RPC_IMAGE_TAG}
+    depends_on:
+      - core
+    ports:
+      - "8000:8000"
+      - "8080:8080"
+    command: --config-path /soroban-rpc.config
+    volumes:
+      - ${RPC_CONFIG_MOUNT_DIR}/stellar-core-integration-tests.cfg:/stellar-core.cfg
+      - ${RPC_CONFIG_MOUNT_DIR}/soroban-rpc.config:/soroban-rpc.config
+      - ${RPC_SQLITE_MOUNT_DIR}:/db/
+    # Needed so that the sql database files created in the container
+    # have the same uid and gid as in the host
```

Comment:
Because the database needs to be accessiable by both the host and the container for the migrations test. Otherwise, after it's created by the container it isn't accessible by the host.